exercism/hamming/hamming.js

```diff
@@ -0,0 +1,27 @@
+var Hamming = function() {}
+
+Hamming.prototype.compute = function (aString, bString) {
+  console.log('we are running the compute function')
+  if(aString == bString) {
```

Comment:
if you do "===" it is a deep equal which will compare type as well which would be a more thorough comparative conditional.